reference/services/git-hosting.rst

```diff
@@ -0,0 +1,90 @@
+Git hosting
+===============
+
+Description
+-----------
+
+Launchpad has direct Git hosting support alongside its existing Bazaar hosting support. 
+The major subsystems involved are:
+
+* The git client
+* Hosting service
+* The underlying object model
+* The web application
+* Repository source code browser 
+
+Git client
+----------
+
+This is what users install on their systems to use Git. 
+It is also installed on the hosting backend to support the ``receive-pack`` and ``upload-pack`` protocols.
+
+Hosting service
+---------------
+
+The outside world connects to the Git hosting service using one of several protocols:
+
+* ``git://``
+* ``git+ssh://``
+* ``https://``
+
+The Git "dumb" HTTP transport will not be supported, as it has largely been considered 
+deprecated since the provision of "smart" HTTP transport in Git v1.6.6 (released January, 2010).
+
+SSL is terminated by haproxy for HTTPS, and all the other protocols also pass through 
```

Comment:
Can we mention which unit this haproxy is?

Reply:
I think it makes more sense to add a link to our mojo-specs repo to understand how haproxy is configured to handle all the protocols that we support, what do you think ?

Reply:
Done!